posthog/tasks/sync_event_and_properties_definitions.py

```diff
@@ -18,11 +19,13 @@ def team_saved(sender: Any, instance: Team, **kwargs: Dict) -> None:
 @app.task(ignore_result=True)
 def sync_event_and_properties_definitions(team_uuid: str) -> None:
 
-    team: Team = Team.objects.only("uuid", *DEFERRED_FIELDS).get(uuid=team_uuid)
+    team: Team = None
```

Comment:
Probably going to fail mypy

Reply:
Oh, you beat me to fixing the types!

Thanks!